lib/compat/wordpress-6.1/script-loader.php

```diff
@@ -181,3 +181,11 @@ function gutenberg_enqueue_global_styles() {
 add_action( 'wp_footer', 'gutenberg_enqueue_global_styles', 1 );
 add_action( 'wp_enqueue_scripts', 'gutenberg_enqueue_stored_styles' );
 add_action( 'wp_footer', 'gutenberg_enqueue_stored_styles', 1 );
+
+function gutenberg_enqueue_classic_theme_styles() {
+	if ( ! wp_is_block_theme() ) {
+		wp_register_style( 'classic-theme-styles', gutenberg_url( 'build/block-library/classic.css' ), array(), true );
+		wp_enqueue_style( 'classic-theme-styles' );
+	}
+}
+add_action( 'after_setup_theme', 'gutenberg_enqueue_classic_theme_styles' );
```

Comment:
This hook throws the following notice for me:

```
Notice: Function wp_register_style was called incorrectly. Scripts and styles should not be registered or enqueued until the wp_enqueue_scripts, admin_enqueue_scripts, or login_enqueue_scripts hooks. This notice was triggered by the classic-theme-styles handle. 
```
I've tested the following - 

`add_action( 'wp_enqueue_scripts', 'gutenberg_enqueue_classic_theme_styles' );`

`add_action( 'init', 'gutenberg_enqueue_classic_theme_styles' );`

- and both work. Not sure which is better. I suppose I'd lean towards `wp_enqueue_scripts` since it's how we enqueue the other styles?

Reply:
> Not sure which is better. I suppose I'd lean towards wp_enqueue_scripts since it's how we enqueue the other styles?

Yes, `wp_enqueue_scripts` would be the right hook to use in this scenario 👍